cms/djangoapps/contentstore/views/course.py

```diff
@@ -829,14 +830,13 @@ def course_info_handler(request, course_key_string):
         if 'text/html' in request.META.get('HTTP_ACCEPT', 'text/html'):
             return render_to_response(
                 'course_info.html',
-                {
+                dict({
                     'context_course': course_module,
                     'updates_url': reverse_course_url('course_info_update_handler', course_key),
                     'handouts_locator': course_key.make_usage_key('course_info', 'handouts'),
                     'base_asset_url': StaticContent.get_base_url_path_for_course_assets(course_module.id),
-                    'keywords_supported': get_keywords_supported(),
                     'push_notification_enabled': push_notification_enabled()
-                }
+                }, **get_context_for_course_info_handler())
```

Comment:
Ideally, every template would have an explicit `context` variable and we could call`dict.update` on it.
To do so here would require changing too much of the surrounding code.

This is the cleanest way for us inject all of our values in one shot.